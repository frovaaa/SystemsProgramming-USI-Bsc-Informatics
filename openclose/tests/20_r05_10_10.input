98115
[2924]
00518
+2107-
02966
50106
49869
{}3<>7<>
49[]2[]
50294
