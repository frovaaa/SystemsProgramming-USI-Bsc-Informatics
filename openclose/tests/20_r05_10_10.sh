#!/bin/bash
exec "$1" "[]{}<>{}\`'+-" < tests/20_r05_10_10.input
