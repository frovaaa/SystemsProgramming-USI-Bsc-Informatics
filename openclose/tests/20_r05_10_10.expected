98115
[2924]
00518
+2107-
02966
50106
49869
37
492
50294
